lua/telescope/_extensions/project.lua

```diff
@@ -0,0 +1,62 @@
+local has_telescope, telescope = pcall(require, 'telescope')
+
+if not has_telescope then
+	error('This plugins requires nvim-telescope/telescope.nvim')
+end
+
+local actions = require("telescope.actions")
+local finders = require("telescope.finders")
+local pickers = require("telescope.pickers")
+local conf = require("telescope.config").values
+local builtin = require("telescope.builtin")
+local project_dirs_file = vim.fn.stdpath('data') .. '/telescope-projects.lua'
```

Comment:
Are you going to define a lua file here? Or will this one be just a txt file

Reply:
I guess just a txt file actually. Problem is I can't figure out how to read a file and import it into a table from the txt file. Do you know how to do this? I need key/value pairs to be imported. I've been attempting to do it but not quite there yet.

Reply:
Commented below. Just not that fast :) I am also done for today and i go to sleep so have fun figuring it out.

Reply:
Me also. Will have a look into it tomorrow. Much appreciated.